test/runtests.jl

```diff
@@ -4,10 +4,12 @@ using SBML
 using SBML: Model, Reaction, Species
 using Symbolics, Unitful
 
+include("symbolics.jl")
```

Comment:
Why outside of the rest of the test suite?

Reply:
it's a library, not a testset

Reply:
ok I guess I solved this. :D